server/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/jeromewu/bookworms/server
```

Comment:
Please help to update this line to below as I forgot to update it during the migration. :sweat_smile: 

```
module github.com/codeday-labs/bookworms/server
```